src/redux/services/api.ts

```diff
@@ -81,13 +83,7 @@ export const api = createApi({
     }),
 
     // PUT: Update customer details
-    updateCustomerById: builder.mutation<
-      Customer,
-      {
-        formData: Partial<AddEditCustomerFormFields>;
-        id: string;
-      }
-    >({
+    updateCustomerById: builder.mutation<Customer, { formData: Partial<AddEditCustomerFormFields>; id: string }>({
```

Comment:
I don't really know if this is something that needs to change, but why is this done? Am I missing something, it looks identical to the previous version?

Reply:
Nothing but formatting changed here. I had a comment about it in the commit description. Maybe it would've been better to have this in a separate commit to avoid confusion...

Reply:
Well, readability suffers quite a lot from the change I would say. Also, if `prettier` would be used it would most likely have split it up. Strange however that it broke your highlighting.

Reply:
Prettier is in the pre-commit hook and it did not try to change this back to what it was.